Source/Core/DolphinWX/Cheats/CheatSearchTab.cpp

```diff
@@ -251,15 +251,33 @@ void CheatSearchTab::UpdateCheatSearchResultItem(long index)
 	m_lview_search_results->SetItem(index, 3, buf);
 }
 
+enum ComparisonMask
+{
+	EQUAL = 0x1,
+	GREATER_THAN = 0x2,
+	LESS_THAN = 0x4
+};
+
+inline ComparisonMask operator | (ComparisonMask comp1, ComparisonMask comp2)
```

Comment:
Don't use inline for functions that are translation-unit scope only. It should be static.